wd_capabilities.au3

```diff
@@ -233,8 +233,21 @@ Func _WD_CapabilitiesAdd($key, $value1 = Default, $value2 = Default)
 		$s_Notation = $_WD_NOTATION__MATCHTYPE & $_WD_NOTATION__SPECIFICVENDOR
 		If $value1 <> '' Then $s_Notation &= '[' & $key & ']'
 
-	Else ; not supported option
-		Return SetError(__WD_Error($sFuncName, $_WD_ERROR_NotSupported, "Not supported KEY parameter ( must be defined in $_WD_KEYS__*** ). " & $s_Parameters_Info))
+	ElseIf _
+			StringRegExp($key, '(?i)' & $_WD_KEYS__STANDARD_PRIMITIVE, $STR_REGEXPMATCH) Or _
+			StringRegExp($key, '(?i)' & $_WD_KEYS__STANDARD_OBJECT, $STR_REGEXPMATCH) Or _
+			StringRegExp($key, '(?i)' & $_WD_KEYS__STANDARD_OBJECT_ARRAY, $STR_REGEXPMATCH) Or _
+			StringRegExp($key, '(?i)' & $_WD_KEYS__SPECIFICVENDOR_PRIMITIVE, $STR_REGEXPMATCH) Or _
+			StringRegExp($key, '(?i)' & $_WD_KEYS__SPECIFICVENDOR_ARRAY, $STR_REGEXPMATCH) Or _
+			StringRegExp($key, '(?i)' & $_WD_KEYS__SPECIFICVENDOR_OBJECT, $STR_REGEXPMATCH) _
+			Then
+		If $_WD_ERROR_MSGBOX Then MsgBox($MB_ICONERROR + $MB_OK + $MB_TOPMOST, @ScriptLineNumber, _
```

Comment:
Why is this here? Doesn't the existing instance in _WD_Error handle this?

Reply:
Fixed: [$_WD_ERROR_MSGBOX removed](https://github.com/Danp2/au3WebDriver/pull/285/commits/b6761ff6be814684bc182f54b7347379adbd7cc0)